### PR TITLE
* Fixes the picturebox alignment

### PR DESCRIPTION
**Pull request by Wagnerp**
image

image

image

### CODE REVIEW

**Comment by Smurf-IV** [comment]
Why is there "SO Much" space around the text ?

**Comment by Smurf-IV** [comment]
Then they need to have null strings to make them "Invisible"
And set the height to be "Auto" as well

**Comment by Wagnerp** [comment]
> Then they need to have null strings to make them "Invisible" And set the height to be "Auto" as well

Fixed sizing issues

Animation

**Comment by Smurf-IV** [comment]
In the demo, When the "Them" option is chosen, it is crammed to the bottom, and does not follow the spacing of the previous intergaps.

**Comment by Wagnerp** [comment]
> In the demo, When the "Them" option is chosen, it is crammed to the bottom, and does not follow the spacing of the previous intergaps.

Adjusted margins to give more space

File: Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonToolkitPoweredByControl.cs [hunk 1]
```diff
@@ -244,6 +244,11 @@ protected override void OnLoad(EventArgs e)
             base.OnLoad(e);
         }
 
+        protected override void OnCreateControl()
```
Comment (Smurf-IV) [inline Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonToolkitPoweredByControl.cs:258]: Why is this here

**Review by Smurf-IV — CHANGES_REQUESTED** [review]

File: Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonToolkitPoweredByControl.cs (hunk 1 above)
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonToolkitPoweredByControl.cs:258]: Was testing sizing. Will be cleaned up in next commit :)
